pkg/cluster/executor/ssh.go

```diff
@@ -176,7 +181,11 @@ func (e *EasySSHExecutor) Execute(cmd string, sudo bool, timeout ...time.Duratio
 // file from remote to local.
 func (e *EasySSHExecutor) Transfer(src string, dst string, download bool) error {
 	if !download {
-		return e.Config.Scp(src, dst)
+		err := e.Config.Scp(src, dst)
+		if err != nil {
+			return errors.Annotatef(err, "failed to scp %s to (%s@%s):%s", src, e.Config.User, e.Config.Server, dst)
```

Comment:
```suggestion
			return errors.Annotatef(err, "failed to scp %s to %s@%s:%s", src, e.Config.User, e.Config.Server, dst)
```
We can reuse ` %s@%s:%s` as out CLI arguments if we want to reproduce manually.